graphium/nn/architectures/global_architectures.py

```diff
@@ -1229,15 +1229,15 @@ def create_module_map(self, level: Union[Literal["layers"], Literal["module"]] =
         if self.task_heads is not None:
             self._module_map.update(
                 {
-                    "graph_output_nn/"
+                    "graph_output_nn-"
```

Comment:
It feels to me that `/` is better since it's less likely to be accidentally mis-used. Why the switch to `-`?

Reply:
I don't know if that's the reason, but hydra doesn't support `/` when overriding from command line (or more precisely `/` token is reserved for another feature). I also had to hack it in the past, so I like the change.

Reply:
Exactly, that's the reason for the change. Thanks @maciej-sypetkowski